apps/api/src/services/sendgrid_handler.py

```diff
@@ -114,3 +124,31 @@ async def send_email(
     except HTTPStatusError as e:
         log.exception("During SendGrid processing: %s", e)
         raise RuntimeError("Could not send email with SendGrid")
+
+
+async def send_decision_email(
+    sender_email: Tuple[str, str], applicant_batch: dict[tuple[str, EmailStr], Decision]
+):
+    accepted_personalization_data = []
+    rejected_personalization_data = []
+    waitlisted_personalization_data = []
```

Comment:
Suggestion: given how we're using three if/elifs below, instead of separate lists, could we create a dictionary with each `Decision` value as the key?